.ci.yaml

```diff
@@ -75,6 +75,99 @@ targets:
       target_file: mac_build_all_plugins.yaml
       channel: stable
 
+  ### iOS tasks ###
+  # TODO(stuartmorgan): Swap this and ios-build_all_plugins once simulator
+  # tests are reliable on the ARM infrastructure. See discussion at
+  # https://github.com/flutter/plugins/pull/5693#issuecomment-1126011089
+  - name: Max_x64 ios_platform_tests_0_4 master
```

Comment:
Is there any plan to building matricies into Cocoon? Having to specify each of these explicitly is much more verbose and error prone than the Cirrus matrix spec. It would also be a huge pain to change the sharding (which we do occasionally) since instead of being able to do a single PR—including the ability to iterate in presubmit to see what the best number of shards is, which is usually what we have done in the past—we'll need to add a whole duplicate set in bringup, land them, see how they go, iterate in post-submit with new PRs if necessary, and then finally do a last PR to switch over.

Reply:
A generic builder will do the trick. It has been planned for release branches (https://github.com/flutter/flutter/issues/97265), and we can do the same for main/master.
But currently it is blocked on LUCI team implementing the support API. Once that's unblocked, we should be able to use cocoon do the scheduling.